discovery-provider/src/queries/queries.py

```diff
@@ -1982,3 +1983,130 @@ def get_top_genre_users():
 
 
         return api_helpers.success_response({ 'user_ids': user_ids })
+
+
+
+# Get the tracks that are 'children' remixes of the requested track
+# The results are sorted by if the original artist has reposted or saved the track
+@bp.route("/remixes/<int:track_id>/children", methods=("GET",))
+def get_remix_tracks(track_id):
```

Comment:
def get_remix_tracks_children?

Reply:
ya, or like `get_remixes_of` to align w our Track metadata names maybe?

Reply:
updated!